package.json

```diff
@@ -16,6 +16,7 @@
     "build": "npm run validate-config && npm run build.app -- --prod",
     "build.e2e": "npm run build.app -- --prod --configuration=e2e",
     "test": "ng test app --code-coverage",
+    "unit": "ng test --browsers=Chrome --watch",
```

Comment:
I think this is going to be a non-standard command? "npm run unit" ?

Reply:
@DenysVuika Yes, indeed. It is nothing even needed, I just found it cumbersome to debug the unit tests (in normal chrome with head) in this repo, that is why I created this shortcut command, it has only development aiding purposes.